Update packaging and service files for SLES 16 (#425)
trento-project/agent: README.md

These could in theory also be installed and configured on other distributions providing the same functionalities, but this use case is not within the scope of the active development.

> In addition to that, _Trento Agent_ also wants the [Prometheus node_exporter component](https://github.com/prometheus/node_exporter) to be running to collect host information for the monitoring functionality.

Reviewer: I would avoid specific systemd lingo here and better articulate that the integration is optional.
```suggestion
In addition to that, _Trento Agent_ can also optionally integrate with the [Prometheus node_exporter component](https://github.com/prometheus/node_exporter) to collect host information for the monitoring functionality.
```

> NOTE: Suggested installation instructions for SUSE-based distributions, adjust accordingly

Install and start `node_exporter` (Optional):

> ```shell

Reviewer: After this code block, we should add another note:

```
> NOTE: the `node_exporter` zypper package might or might not be available depending on the SLES version.
```
